pkg/graveler/graveler.go

```diff
@@ -1744,11 +1745,7 @@ func (g *Graveler) safeBranchWrite(ctx context.Context, log logging.Logger, repo
 		branchPreOp *Branch
 	)
 	defer func() {
-		retries := try
-		if retries > options.MaxTries {
-			retries = options.MaxTries
-		}
```

Comment:
How do you enforce this?

Reply:
I don't, I don't want to, and this is more like a mistake: it counts the number of times that we tried, rather than the number of times that we failed.  If `retries > options.MaxTries` then we failed too many times and decided to give up.  This means that we actually failed one extra time.

Technically this is the same number of retries as if the last time succeeded.  But I think this count is possibly nicer.  But I have no firm opinion on this, and will revert this change if you prefer.